src/components/news-item.vue

```diff
@@ -27,20 +27,15 @@
     <template v-else>
       <ion-card class="post-text ion-padding">
         {{ $t("activity.unknownType") }}
-        <ion-grid>
-          <ion-row>
-            <ion-col>
-              <ion-button fill="outline" :href="androidLink">{{
-                $t("appPage.button.android")
-              }}</ion-button>
-            </ion-col>
-            <ion-col>
-              <ion-button fill="outline" :href="iosLink">{{
-                $t("appPage.button.iOS")
-              }}</ion-button>
-            </ion-col>
-          </ion-row>
-        </ion-grid>
+        <ion-button v-if="isPlatform('android')" fill="outline" :href="androidLink">{{
```

Comment:
I suppose this will be called every time we render this way. iiuc vue generally prefers to put that into `computed` properties. and considering that we are reusing these we might want to put those into a shared utils?

```typescript
# src/utils/platform.ts

import { computed } from "vue";
import { isPlatform } from "@ionic/vue";

export {
  isAndroid: computed(() => isPlatform('android'),
  isIos: computed(() => isPlatform('ios'),
}
```